v5/heroku.go

```diff
@@ -15,18 +15,17 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/google/go-querystring/query"
 	"io"
 	"net/http"
 	"reflect"
 	"runtime"
 	"strings"
 	"time"
-
-	"github.com/google/go-querystring/query"
 )
 
 const (
-	Version          = "v5"
+	Version          = "v3"
```

Comment:
`v5` was correct before. This value needs to track the semantic version, not the Heroku API version, which is set in a static string in [`transport.go`](https://github.com/heroku/heroku-go/blob/master/v5/transport.go#L74).

Reply:
That's surprising to me; the constant is definitely intended to reflect the API version, at least in the general case for schematic generated systems. If we want it to be different, we'll need to update the [generation script](https://github.com/heroku/heroku-go/blob/master/script/generate#L41) which explicitly sets the version to v3.

Reply:
In the `DefaultUserAgent` string on the next line, this is intended to reflect the current version of the Go client.

Since the actual version identifier in [`transport.go`](https://github.com/heroku/heroku-go/blob/master/v5/transport.go#L74) is `3`, not `v3`, it doesn't seem like this value was intended for that purpose.

Is it likely folks use `heroku.Version` directly? Would they think this means the Heroku API version or the module's semantic version? 😅

Reply:
Oh, interesting. Well, sounds like we need to fix the generation script to set v5 then! 👍

Reply:
I fixed the `generate` script to output v5 but rolled it back since I thought we wanted to use the actual API version. I can unroll that change back.

Reply:
This version is the semantic Go Mod version, not the Heroku API version, so please unroll it back @davidji99 😅

Reply:
Sounds good. I already unrolled it back @mars.